src/helpers.ts

```diff
@@ -0,0 +1,12 @@
+import moviesFromServer from './api/movies.json';
+
+export const showFilms = (query: string) => moviesFromServer.filter(movie => {
+  const movieTitleInLowerCase = movie.title.toLowerCase();
+  const queryInLowerCase = query.toLowerCase();
+  const descriptionInLowerCase = movie.description.toLowerCase();
+  const isRightTitle = movieTitleInLowerCase.includes(queryInLowerCase);
+  const isRightDescription = descriptionInLowerCase
+    .includes(queryInLowerCase);
+
+  return isRightTitle || isRightDescription;
+});
```

Comment:
`const queryInLowerCase = query.toLowerCase();` will be created every iteration in filter.
Move it to component and provide as argument
Use destructuring assignment to improve readability.
You can chain some methods to avoid bulkiness

```suggestion
export const getMoviesByQuery = (query: string) => moviesFromServer.filter(movie => {
  const { title , description } = movie;
  
  const isRightTitle = title.toLowerCase().includes(query);
  const isRightDescription = description.toLowerCase().includes(query);

  return isRightTitle || isRightDescription;
});
```